packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/EditContract.jsx

```diff
@@ -102,24 +108,24 @@ const EditContract = props => {
     })
   }
 
-  const [shortLabel, setShortLabel] = useState(getAccountLabel(account))
-  const [owners, setOwners] = useState(getAccountOwners(account))
+  const [shortLabel, setShortLabel] = useState(getAccountLabel(contract))
+  const [owners, setOwners] = useState(getAccountOwners(contract))
 
   const handleSubmit = e => {
     e.preventDefault()
-    handleSaveAccount(account, { shortLabel, owners })
+    handleSaveContract(contract, { shortLabel, owners })
   }
 
   const handleRequestDeletion = ev => {
     ev.preventDefault()
     setShowDeleteConfirmation(true)
   }
 
-  const handleRemoveAccount = async account => {
+  const handleRemoveAccount = async contract => {
```

Comment:
handleRemoveContract ?

Reply:
Thanks, done in c79ce85d90a8bc508c7956d348426e9907f3a3b4.